From dac5220198c538c73955ce3540b3981e05f42a22 Mon Sep 17 00:00:00 2001
From: Marcel Bargull <marcel.bargull@udo.edu>
Date: Sun, 16 Oct 2022 06:45:10 +0200
Subject: [PATCH] Replace PROBLEM with Rf_error

---
 src/fexact.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/fexact.c b/src/fexact.c
index 85d6a8f..f1ca108 100644
--- a/src/fexact.c
+++ b/src/fexact.c
@@ -1022,7 +1022,7 @@ LoopNode: /* Generate a node */
 	    key = it[i] + key * kyy;
 	}
 	if (key < -1)
-	    PROBLEM "Bug in FEXACT: gave negative key" RECOVER(NULL_ENTRY);
+	    Rf_error("Bug in FEXACT: gave negative key");
 	/* Table index */
 	ipn = key % ldst + 1;
 	/* Find empty position */
@@ -1803,7 +1803,7 @@ void prterr(int icode, const char *mes)
      mes    - Character string containing the error message.	(Input)
   -----------------------------------------------------------------------
   */
-    PROBLEM "FEXACT error %d.\n%s", icode, mes RECOVER(NULL_ENTRY);
+    Rf_error("FEXACT error %d.\n%s", icode, mes);
     return;
 }
 
-- 
2.38.0

